File: source/utils/nodemon.py
from io import open
from os import path


def nodemon():
    nodemon_file = 'nodemon.json'

    nodemon_file_content = '''
\u007b
  "watch": [
    "src"
  ],
  "ext": "ts",
  "ignore": [
    "src/**/*.test.ts"
  ],
  "exec": "ts-node -r dotenv/config ./src/index"
\u007d
'''

    with open(nodemon_file, 'w') as f:
        f.write(nodemon_file_content)

File: lib/src/functions/nodemon.js
const writeFile = require('../utils/writeFile')

module.exports = async () => {
  const data = {
    nodemonContent: `{
  "watch": [
    ".env",
    "src"
  ],
  "ext": "ts",
  "ignore": [
    "src/**/*.test.ts"
  ],
  "exec": "npx ts-node -r dotenv/config ./src/index"
}
`,
    nodemonFile: 'nodemon.json'
  }

  await writeFile(data.nodemonFile, data.nodemonContent)
}
